server/sonar-web/src/main/js/apps/permission-templates/views/UsersView.js

```diff
@@ -17,14 +17,37 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-import _ from 'underscore';
 import Modal from '../../../components/common/modals';
 import Template from '../templates/permission-templates-users.hbs';
 import '../../../components/SelectList';
+import {
+    addProjectCreatorToTemplate,
+    removeProjectCreatorFromTemplate
+} from '../../../api/permissions';
 
 export default Modal.extend({
   template: Template,
 
+  events () {
+    return {
+      ...Modal.prototype.events.apply(this, arguments),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use the rest syntax to declare this function's arguments. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS3513)